spec/controllers/miq_policy_controller/rsop_spec.rb

```diff
@@ -0,0 +1,28 @@
+include UiConstants
```

Comment:
@Fryguy Sadly these are already included globally, on purpose, at their origin. In any case, minimized confusion in #6585